### PR TITLE
New tee type: CSV (China Secure Virtualization)

### PR DESCRIPTION
**Pull request by BaoshunFang**

### CODE REVIEW

**Comment by jialez0** [comment]
@sameo @slp @Could you please take a look at this PR? Thanks a lot!

File: src/lib.rs [hunk 1]
```diff
@@ -17,6 +17,7 @@ pub enum Tee {
     Tdx,
     // Arm Confidential Compute Architecture
     Cca,
+    Csv,
```
Comment (slp) [inline src/lib.rs:20]: Could you please also add a comment with the meaning of the acronym, as we have above for Cca?

**Review by slp — CHANGES_REQUESTED** [review]

File: src/lib.rs (hunk 1 above)
Comment (BaoshunFang) [reply src/lib.rs:21]: Done

**Review by slp — APPROVED** [review]
LGTM, thanks!

**Review by tylerfanelli — APPROVED** [review]

**Pull request merged**